ci/terraform/oidc/vpc-peering.tf

```diff
@@ -0,0 +1,51 @@
+resource "aws_vpc_peering_connection" "vpc_peering" {
```

Comment:
I wonder if this should live in di-infrastructure instead, next to the VPC definition?